style-guide/code-style/html/README.md

```diff
@@ -0,0 +1,6 @@
+# HTML 
+
+## Formatting
+* Don't use inline styles, unless you are generating dynamic styles in your templating system.
```

Comment:
We should always start with the most maintainable thing until/unless there's a specific need for some sort of gnarly optimization. I think @oliverthomasklein is trying to write general guidelines.